src/main/resources/assets/caverns_and_chasms/lang/ru_ru.json

```
@@ -0,0 +1,810 @@
+  "subtitles.caverns_and_chasms.block.lava_lamp.glug": "Лавовая лампа бурлит",
+  "subtitles.caverns_and_chasms.block.tin.deflect": "Снаряд отскакивает",
+  "subtitles.caverns_and_chasms.block.toolbox.close": "Ящик с инструментами закрывается",
+  "subtitles.caverns_and_chasms.block.toolbox.open": "Ящик с инструментами открывается",
+  "subtitles.caverns_and_chasms.effect.rewind.rewind": "Что-то возвращается",
+  "subtitles.caverns_and_chasms.entity.parrot.imitate.deeper": "Попугай шипит",
+  "subtitles.caverns_and_chasms.entity.parrot.imitate.mime": "Попугай имитирует",
+  "subtitles.caverns_and_chasms.entity.parrot.imitate.peeper": "Попугай шипит",
+  "subtitles.caverns_and_chasms.item.armor.equip_horse_necromium": "Лязг некромиевой конской брони",
+  "subtitles.caverns_and_chasms.item.armor.equip_horse_netherite": "Лязг незеритовой конской брони",
+  "subtitles.caverns_and_chasms.item.armor.equip_necromium": "Лязг некромиевой брони",
+  "subtitles.caverns_and_chasms.item.armor.equip_sanguine": "Бряцанье сангвинической брони",
+  "subtitles.caverns_and_chasms.item.armor.equip_silver": "Бряцанье серебряной брони",
+  "subtitles.caverns_and_chasms.item.bejeweled_pearl.crumble": "Украшенная жемчужина рассыпается",
+  "subtitles.caverns_and_chasms.item.bejeweled_pearl.teleport": "Кто-то телепортируется",
+  "subtitles.caverns_and_chasms.item.copper_horn.play": "Медный рог играет",
+
+  "advancements.caverns_and_chasms.adventure.atone_item.description": "Зачаруйте предмет на искупленческом столе, пожертвовав его прочностью",
+  "advancements.caverns_and_chasms.adventure.atone_item.title": "Неравноценный обмен",
+  "advancements.caverns_and_chasms.adventure.bejeweled_anvil_repair.description": "Модифицируйте или почините предмет на украшенной наковальне и посмотрите, что будет",
+  "advancements.caverns_and_chasms.adventure.bejeweled_anvil_repair.title": "Обескураженный",
+  "advancements.caverns_and_chasms.adventure.break_item_atoning.description": "Сломайте предмет, заигравшись с искупленческим столом",
+  "advancements.caverns_and_chasms.adventure.break_item_atoning.title": "Одержимый",
+  "advancements.caverns_and_chasms.adventure.dismantle_item.description": "Успешно демонтируйте ваше снаряжение на столе демонтажа, чтобы вернуть потраченный кузнечный шаблон",
+  "advancements.caverns_and_chasms.adventure.dismantle_item.title": "Без фокусов",
+  "advancements.caverns_and_chasms.adventure.dont_move.description": "Оно следит за тобой",
+  "advancements.caverns_and_chasms.adventure.dont_move.title": "Замри!",
+  "advancements.caverns_and_chasms.adventure.repair_with_zirconia.description": "Почините предмет с помощью фианита на украшенной наковальне",
+  "advancements.caverns_and_chasms.adventure.repair_with_zirconia.title": "Приукрашенный",
+  "advancements.caverns_and_chasms.nether.ride_boat_with_deeper.description": "Вместе с дипером, сядьте в лодку и телепортируйтесь в Незер",

```

Чтобы чуть менее по-английски:
```suggestion
  "advancements.caverns_and_chasms.nether.ride_boat_with_deeper.description": "Сядьте в лодку вместе с дипером и телепортируйтесь в Незер",
```